src/App.js

```diff
@@ -1,18 +1,39 @@
+import React,{useState} from "react";
 import DiaryCard from "./Components/DiaryCard";
-import DiaryHome
- from "./Components/DiaryHome";
+import DiaryHome from "./Components/DiaryHome";
+
+
 function App() {
+
+
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [card,setCard] =useState(false);
+ 
+  function onClick(e) {
+    e.preventDefault();
+
+      if(title.trim().length === 0 || description.trim().length ===0){
+        console.log("Missing title or Missing Description");
+      }
+      else{
+        setCard(true);
+      }
```

Comment:
update the state with entered values and use proper data structure to store multiple cards.